framework/wazuh/core/engine/events.py

```diff
@@ -1,13 +1,17 @@
+from dataclasses import asdict
+from httpx import RequestError
 from typing import List
 
-from wazuh.core.engine.base import BaseModule
+from wazuh.core.engine.base import APPLICATION_JSON, BaseModule, convert_enums
+from wazuh.core.engine.models.base import ErrorResponse
 from wazuh.core.engine.models.events import StatelessEvent
+from wazuh.core.exception import WazuhEngineError
 
 
 class EventsModule(BaseModule):
     """Events module to send stateless events to the Engine."""
 
-    MODULE = 'events'
+    MODULE = 'events/stateless'
```

Comment:
I would call it simply `events` although it is currently used only for stateless events. The reason being that it is more concise for things like logs or metrics from this module. e.g `[Engine] [<module>] INFO: <message>`